tests/unit/lms/services/canvas_api/factory_test.py

```diff
@@ -48,14 +43,13 @@ def test_building_the_AuthenticatedClient(
     def BasicClient(self, patch):
         return patch("lms.services.canvas_api.factory.BasicClient")
 
-    @pytest.fixture(autouse=True)
-    def TokenStore(self, patch):
-        return patch("lms.services.canvas_api.factory.TokenStore")
-
     @pytest.fixture(autouse=True)
     def AuthenticatedClient(self, patch):
         return patch("lms.services.canvas_api.factory.AuthenticatedClient")
 
     @pytest.fixture(autouse=True)
     def CanvasAPIClient(self, patch):
         return patch("lms.services.canvas_api.factory.CanvasAPIClient")
+
+
+pytestmark = pytest.mark.usefixtures("ai_getter", "token_store_service")
```

Comment:
Let's use these module level `pytestmark`'s for activating service fixtures in a test module. It's what all our other test modules use to activate other service fixtures, and it activates it for the entire test module not just one class or test like the `@pytest.mark.usefixtures` decorator. It may seem more efficient to try to activate the fixture only for those tests that need it, but in practice this saves little, adds noise  and easily leads to mistakes where a test that should use the fixture doesn't. It's just easier to slap one of these at the bottom of every test file that needs services